Arm/joint.hpp

```diff
@@ -53,9 +42,10 @@ class Joint
   }
 
   /// Move the motor to the (calibrated) angle desired.
-  void SetPosition(units::angle::degree_t angle)
+  void SetPosition(double angle)
   {
-    units::angle::degree_t calibrated_angle = angle - zero_offset_angle;
+    units::angle::degree_t angle_to_degrees(angle);
+    units::angle::degree_t calibrated_angle = angle_to_degrees - zero_offset_angle;
     calibrated_angle                        = units::math::min(
         units::math::max(calibrated_angle, minimum_angle), maximum_angle);
```

Comment:
is this logic for making sure joint doesn't go beyond max or min angles?
could be rewritten using std::clamp e.g.:
```suggestion
    calibrated_angle = clamp(calibrated_angle, minimum_angle, maximum_angle); 
```

Reply:
oh but it might freak out about the types since clamp might not work nicely with units library

Reply:
I might have to redo quite a bit of the joint class, I didn't write that function so I'm trying to understand/edit what I can. I have no idea what the last guy was attempting to do there.